app/src/main/java/net/gsantner/markor/frontend/search/Occurrence.java

```diff
@@ -0,0 +1,58 @@
+package net.gsantner.markor.frontend.search;
+
+public class Occurrence {
+    public static final int BACKGROUND_COLOR = 0x90FF0000;
+    public static final int SPECIAL_BACKGROUND_COLOR = 0x90FFA500;
+    private BackgroundColorSpan backgroundColorSpan;
+
+    private int startIndex;
+    private int endIndex;
+
+    public BackgroundColorSpan createBackgroundColorSpan() {
+        this.backgroundColorSpan = new BackgroundColorSpan(BACKGROUND_COLOR);
+        return this.backgroundColorSpan;
+    }
+
+    public BackgroundColorSpan createSpecialBackgroundColorSpan() {
```

Comment:
Something to watch out for - having spans which are not handled with the highlighter may cause performance issues, especially with larger files